_redirects

```diff
@@ -10,6 +10,8 @@
 # e.g: /components/sensors/abc.html /components/sensors/xyz.html 301
 /components/sensor/sgp40.html /components/sensor/sgp4x.html 301
 /components/binary_sensor/xpt2046.html /components/touchscreen/xpt2046.html 301
+/components/esp32_camera.html /components/camera/esp32_camera.html 301
+/components/esp32_camera_web_server.html /components/camera/esp32_camera_web_server.html 301
```

Comment:
While this makes logical sense, since camera is not a `PLATFORM_COMPONENT` and `esp32_camera` is not a platform of that component, the pages should stay where they are.
That is how we try to keep structure to the `components` folder for docs.